bundler.go

```diff
@@ -413,6 +421,11 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		}
 	}
 
+	if b.buildTags != "" {
```

Comment:
I'd rather build the args before the `// Build cmd` comment. That way you don't need to play with `cmd.Args` indexes directly.

All you'd have to add is something along the lines of:

```go
// TODO Init args
var args []string

// Loop through build flags
for k, v := range b.buildFlags {
  args = append(args, []string{"-"+k, v}...)
}

// TODO Finish building args
```